### DISCUSSION TITLE
Alpine.morph seems confused by <br>

### DISCUSSION THREAD

**Discussion by earthboundkid** [alpinejs/alpine#2463] [7. Bugs (archived)] [answered]
I have text like this:

```html
<h2 class="text-center text-2xl font-semibold" x-template>
  ${age} ${kind} <br> District ${props.district}
</h2>
```

I'm trying to morph it to

```
<h2 class="text-center text-2xl font-semibold" x-template>
   Old Senate <br> District 1
</h2>
```

But I am getting

```
<h2 class="text-center text-2xl font-semibold" x-template>
   Old Senate <br> District ${props.district}
</h2>
```

Seems like a 🐛 .

Link to commit with the bug: https://github.com/spotlightpa/viz-redistricting-2020/commit/dd587fa444d8c9fd6e3217aafacbc2b58920f29b

**Comment by earthboundkid** [comment]
It works if I change the template to `${age} ${kind} <br> <span>District ${props.district}</span>`. I think it just decides not to update the final text fragment.

**Comment by SimoTod** [comment, marked as answer]
It should be fixed by #2524
